### ISSUE TITLE
[QueryBuilder] `*Batch()` methods are inconsistant

### ISSUE THREAD

**Issue by kenjis** [codeigniter4/CodeIgniter4#6315]
### `set*Batch()`

```php
public function setInsertBatch($key, string $value = '', ?bool $escape = null);
public function setUpdateBatch($key, string $index = '', ?bool $escape = null);
```

`setUpdateBatch()` does not have `$value`.

### `*Batch()`

```php
public function insertBatch(?array $set = null, ?bool $escape = null, int $batchSize = 100);
public function updateBatch(?array $set = null, ?string $index = null, int $batchSize = 100);
```

`updateBatch()` does not have `$escape`.

### References

- #5134
- #5148
- #5674
- #6153
- #6294

**Comment by sclubricants** [comment]
Related: https://github.com/codeigniter4/CodeIgniter4/pull/6407/ https://github.com/codeigniter4/CodeIgniter4/issues/5149

**Comment by kenjis** [comment]
Closed by #6536
